concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1445,6 +1448,39 @@ jobs:
 
                   cf push
 
+        # FIXME: Remove once run on production
+        - task: fix-system-domain-org
+          config:
+            platform: linux
+            image: docker:///governmentpaas/cf-cli
+            inputs:
+              - name: paas-cf
+              - name: bosh-CA
+              - name: config
+            params:
+            run:
+              path: sh
+              args:
+                - -e
+                - -u
+                - -c
+                - |
+                  ./paas-cf/concourse/scripts/import_bosh_ca.sh
+                  . ./config/config.sh
+
+                  echo | cf login -a "${API_ENDPOINT}" -u "${CF_ADMIN}" -p "${CF_PASS}"
+
+                  if cf org GDS > /dev/null; then
+                    cf delete-org GDS -f
```

Comment:
what happens to domain when you do this? what happens to apps using that domain?